src/forms/inputs.js

```diff
@@ -7,16 +7,16 @@ import {
 } from 'semantic-ui-react';
 
 
-/** A big button. Right now, just used in the bottom row of
-*     form sections
+// @todo Change ...buttonProps to explicit `overrides` obj
+/** A big button with overridable properties.
 *
 * @function
 * @param {object} props
-* @property {buttonProps} props.children - React children
-* @property {buttonProps} props.buttonProps - Props to
+* @property {buttonProps} props.children Things React can render
+* @property {buttonProps} props.buttonProps Props to
```

Comment:
Yeah, this component needs to change anyway. It might be better to delete the description altogether... Yeah, I think that would be best for cleaning.